--- create_oq_schema.orig	2013-02-12 19:09:17.000000000 +0100
+++ create_oq_schema	2013-02-12 19:16:21.000000000 +0100
@@ -32,7 +32,7 @@
 psql_batch_options='--set ON_ERROR_STOP=1'
 
 # Where do the table spaces live?
-tspace_path='/var/lib/postgresql/9.1/main/ts'
+tspace_path='/home/openquake/local/var/postgtresql/ts'
 # What table spaces should be created?
 tspace_list="admin eqcat hzrdi hzrdr oqmif riski riskr uiapi htemp"
 
@@ -201,13 +201,13 @@
 psql -p $db_port -d postgres -U $db_admin_user -c "CREATE DATABASE $db_name"
 createlang -p $db_port -d $db_name -U $db_admin_user plpythonu
 
-postgis="/usr/share/postgresql/9.1/contrib/postgis-1.5"
+postgis="/home/openquake/local/share/postgresql/contrib/postgis-2.0"
 # Load the PostGIS stuff into the newly created OpenQuake database.
 echo ".. Loading postgis functions/data into $db_name .."
 psql -p $db_port -U $db_admin_user $psql_batch_options -d $db_name -f $postgis/postgis.sql
 
 psql -p $db_port -U $db_admin_user $psql_batch_options -d $db_name -f $postgis/spatial_ref_sys.sql
-psql -p $db_port -U $db_admin_user $psql_batch_options -d $db_name -f /usr/share/postgresql/9.1/contrib/postgis_comments.sql
+psql -p $db_port -U $db_admin_user $psql_batch_options -d $db_name -f $postgis/postgis_comments.sql
 
 echo ".. Running schema definition file: $schema_file .."
 psql -p $db_port -U $db_admin_user $psql_batch_options -d $db_name -f $schema_file
