--- ../openquake/engine/bin/oq_create_db        2014-07-08 18:09:27.191999982 +0200
+++ oq_create_db        2014-07-09 10:06:54.413999387 +0200
@@ -30,7 +30,7 @@
 psql_batch_options='--set ON_ERROR_STOP=1'

 # Where do the table spaces live?
-tspace_path='/var/lib/postgresql/9.1/main/ts'
+tspace_path='_OQPREFIX_/local/var/postgresql/ts'
 # What table spaces should be created?
 tspace_list="admin hzrdi hzrdr riski riskr uiapi htemp"

@@ -43,7 +43,7 @@
 verbose="off"
 fixtures=""

-db_admin_user="postgres"
+db_admin_user="_OQUSER_"

 this_is_linux=0
 # Are we running on linux?
@@ -197,14 +197,13 @@
 psql -p $db_port -d postgres -U $db_admin_user -c "CREATE DATABASE $db_name" $psql_batch_options
 createlang -p $db_port -d $db_name -U $db_admin_user plpythonu

-postgres_contrib="/usr/share/postgresql/9.1/contrib"
-postgis="$postgres_contrib/postgis-1.5"
+postgis="_OQPREFIX_/local/share/postgresql/contrib/postgis-1.5"
 # Load the PostGIS stuff into the newly created OpenQuake database.
 echo ".. Loading postgis functions/data into $db_name .."
 psql -p $db_port -U $db_admin_user $psql_batch_options -d $db_name -f $postgis/postgis.sql

 psql -p $db_port -U $db_admin_user $psql_batch_options -d $db_name -f $postgis/spatial_ref_sys.sql
-psql -p $db_port -U $db_admin_user $psql_batch_options -d $db_name -f $postgres_contrib/postgis_comments.sql
+psql -p $db_port -U $db_admin_user $psql_batch_options -d $db_name -f $postgis/postgis_comments.sql

 echo ".. Running schema definition file: $schema_file .."
 if [ "$verbose" = "off" ]; then
