--- ../openquake/engine/bin/oq_create_db	2014-09-09 16:48:01.999342858 +0200
+++ oq_create_db	2014-09-09 16:51:03.017707647 +0200
@@ -29,7 +29,7 @@
 psql_batch_options='--set ON_ERROR_STOP=1'
 
 # Where do the table spaces live?
-tspace_path='/var/lib/postgresql/9.1/main/ts'
+tspace_path='_OQPREFIX_/local/var/postgresql/ts'
 # What table spaces should be created?
 tspace_list="hzrdi hzrdr riski riskr uiapi"
 
@@ -41,7 +41,7 @@
 verbose="off"
 fixtures=""
 
-db_admin_user="postgres"
+db_admin_user="_OQUSER_"
 
 # What is the name of the db group that all OpenQuake db users belong to?
 db_group_name="openquake"
@@ -169,13 +169,12 @@
 echo ".. Creating database $db_name .."
 psql -p $db_port -d postgres -U $db_admin_user -c "CREATE DATABASE $db_name OWNER oq_admin" $psql_batch_options
 
-postgres_contrib="/usr/share/postgresql/9.1/contrib"
-postgis="$postgres_contrib/postgis-1.5"
+postgis="_OQPREFIX_/local/share/postgresql/contrib/postgis-1.5"
 # Load the PostGIS stuff into the newly created OpenQuake database, must be run as postgres
 echo ".. Loading postgis functions/data into $db_name .."
 psql -p $db_port -U $db_admin_user $psql_batch_options -d $db_name -f $postgis/postgis.sql
 psql -p $db_port -U $db_admin_user $psql_batch_options -d $db_name -f $postgis/spatial_ref_sys.sql
-psql -p $db_port -U $db_admin_user $psql_batch_options -d $db_name -f $postgres_contrib/postgis_comments.sql
+psql -p $db_port -U $db_admin_user $psql_batch_options -d $db_name -f $postgis/postgis_comments.sql
 
 # fix ownership of three public tables from postgres to oq_admin
 psql -p $db_port -U $db_admin_user $psql_batch_options -d $db_name -c "ALTER TABLE spatial_ref_sys OWNER TO oq_admin"
